plugins/catalog-backend-module-bitbucket/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@backstage/plugin-catalog-backend-module-bitbucket",
+  "description": "A Backstage catalog backend module that helps integrate towards Bitbucket",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": false,
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "backend-plugin-module"
+  },
+  "homepage": "https://backstage.io",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/backstage/backstage",
+    "directory": "plugins/catalog-backend-module-bitbucket"
+  },
+  "keywords": [
+    "backstage"
+  ],
+  "scripts": {
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack",
+    "clean": "backstage-cli package clean",
+    "start": "backstage-cli package start"
+  },
+  "dependencies": {
+    "@backstage/backend-common": "^0.13.0-next.0",
```

Comment:
I saw that your last commit changed some dependencies to the pre-release versions. Will they be changed to the release version? (before merging/later after merging/...)

Reply:
So the entire repo has now entered "pre" mode (see https://github.com/backstage/backstage/pull/10083) which means that it's statefully now all about publishing `-next.X` versions. You'll see that there's a mixture of `next` and non-`next` versions, based on which packages have actually had changes since pre was entered, and that's fine too. When we leave "pre" mode as a dedicated action, it's to prepare for a non-`next` release.

Reply:
This may seem odd if you're used to multi-branch development with one branch per version, but it's an intentional choice we made for the time being, to be able to maintain a very high rate of development and change without being too bogged down in release management. This may be reconsidered in the far future if it's deemed necessary, but for now we feel that this is a neat compromise. We still have a hot-patch flow as well that we can leverage in the case of botched releases or security reports.

Reply:
thanks for explaining